src/components/shared/articleContainer.tsx

```diff
@@ -1,102 +1,109 @@
-    <style>
-      @font-face {
+import React from 'react';
+import { css } from '@emotion/core';
+
+const ArticleContainerStyles = css`
+    @font-face {
         font-family: "Guardian Egyptian Web";
         src: url('/public/fonts/egyptiantext-regular.otf');
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Egyptian Web";
         font-weight: 500;
         src: url("/public/fonts/egyptiantext-medium.otf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Egyptian Web";
         font-weight: bold;
         src: url("/public/fonts/egyptiantext-medium.otf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Text Sans Web";
         src: url('/public/fonts/GuardianSansWeb-Regular.ttf');
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Headline";
         font-weight: 200;
         src: url("/public/fonts/GHGuardianHeadline-Light.ttf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Headline";
         font-weight: 400;
         src: url("/public/fonts/GHGuardianHeadline-Regular.ttf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Headline";
         font-weight: 500;
         src: url("/public/fonts/GHGuardianHeadline-Medium.ttf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Headline";
         font-weight: 700;
         src: url("/public/fonts/GHGuardianHeadline-Bold.ttf");
-      }
+    }
 
-      @font-face {
+    @font-face {
         font-family: "Guardian Icons";
         src: url('/public/fonts/icons.otf');
-      }
+    }
 
-      html {
-          font-size: 62.5%;
-          background: white;
-      }
+    font-size: 62.5%;
+    background: white;
 
-      body {
+    body {
         margin: 0;
         font-family: 'Guardian Egyptian Web';
         font-size: 1.6em;
         line-height: 1.5em;
         overflow-x: hidden;
-      }
+    }
 
-      figure {
+    figure {
         margin: 1em 0;
-      }
+    }
 
-      figure.element-embed {
+    figure.element-embed {
         margin: 2em 0
-      }
+    }
 
-      .js-email-sub__iframe {
+    .js-email-sub__iframe {
         width: 100%;
-      }
+    }
 
-      .js-email-sub__iframe + figcaption {
+    .js-email-sub__iframe + figcaption {
         margin-top: -8px;
-      }
+    }
 
-      video,
-      .element-atom iframe,
-      .element-audio iframe {
+    video,
+    .element-atom iframe,
+    .element-audio iframe {
         width: 100%;
-      }
-    </style>
-</head>
-<body>
-  <div id="root"></div>
-  <script src="/assets/client.js"></script>
-  <script src="https://platform.twitter.com/widgets.js"></script>
-</body>
-</html>
\ No newline at end of file
+    }
+`;
+
+function ArticleContainer(includesTweets: boolean): JSX.Element {
```

Comment:
Should the logic that decides whether to include tweets or not be part of the component rather than outside?

Reply:
(I came to that conclusion after reading this excellent blog post: https://www.theguardian.com/info/2019/dec/08/migrating-the-guardian-website-to-react )

Reply:
Good point, it's only used in that one place so makes sense to be moved inside